stone/target/js_client.py

```diff
@@ -24,6 +26,13 @@
           'contains all of the routes.'),
 )
 _cmdline_parser.add_argument(
+    'typedef_filename',
```

Comment:
Also, perhaps we should use the same pattern that's used for python and swift. That is, we have two generators: `python_types` and `python_client`. The former is invoked once to generate all types (in this case, we would just generate the typedefs). The latter is invoked twice (once for user routes, once for team routes).